fs/zbd_zenfs.cc

```diff
@@ -204,8 +204,11 @@ static std::string write_latency_metric_name = "zenfs_write_latency";
 static std::string read_latency_metric_name = "zenfs_read_latency";
 static std::string sync_latency_metric_name = "zenfs_sync_latency";
 static std::string io_alloc_latency_metric_name = "zenfs_io_alloc_latency";
-static std::string io_alloc_actual_latency_metric_name =
-    "zenfs_io_actual_alloc_latency";
+static std::string io_alloc_wal_latency_metric_name = "zenfs_io_alloc_wal_latency";
+static std::string io_alloc_non_wal_latency_metric_name = "zenfs_io_alloc_non_wal_latency";
+static std::string io_alloc_wal_actual_latency_metric_name = "zenfs_io_alloc_wal_actual_latency";
+static std::string io_alloc_non_wal_actual_latency_metric_name = "zenfs_io_alloc_non_wal_actual_latency";
+static std::string io_alloc_actual_latency_metric_name = "zenfs_io_actual_alloc_latency";
 static std::string meta_alloc_latency_metric_name = "zenfs_meta_alloc_latency";
```

Comment:
I think we don't need the previous alloc_latency & actual_alloc_latency anymore

Reply:
Thanks! I removed the alloc_latency & actual_alloc_latency, please check again.